README.md

```diff
@@ -1,303 +1,37 @@
-# Microsoft Bot Framework Web Chat
+# botframework-webchat
 
-Embeddable web chat control for the [Microsoft Bot Framework](http://www.botframework.com) using the [DirectLine](https://docs.botframework.com/en-us/restapi/directline3/) API.
+[![npm version](https://badge.fury.io/js/botframework-webchat.svg)](https://badge.fury.io/js/botframework-webchat) [![Build Status](https://travis-ci.org/Microsoft/BotFramework-WebChat.svg?branch=master)](https://travis-ci.org/Microsoft/BotFramework-WebChat)
 
-Used by the Bot Framework developer portal, [Emulator](https://github.com/Microsoft/BotFramework-Emulator), Web Chat channel, and [Azure Bot Service](https://azure.microsoft.com/en-us/services/bot-service/)
+# Roadmap
```

Comment:
Generally this page should be cleaned up. Something like:

This is the preview version of the v4 Microsoft Bot Framework Web Chat. It does XYZ and allows to XYZ. This chat client is used with the Microsoft Azure Bot etc etc..

The main difference between the v4 and v3 Microsoft Bot Framework Web Chat is XYZ.

What is Parity... are these features that are missing or are done? What is chores? Instead that should be todo items or something.

Need samples, docs, etc... much more like it was before.